### PR TITLE
Fix bug 1139490: Switch to django-pipeline

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

File: templates/includes/config.html [hunk 1]
```diff
@@ -31,14 +30,15 @@
...
             assets: {
                 css: {
-                    'wiki-compat-tables': {{ css_url_array('wiki-compat-tables') }}
+                    'editor-content': [{% stylesheet 'editor-content' %}],
+                    'wiki-compat-tables': [{% stylesheet 'wiki-compat-tables' %}]
                 },
                 js: {
-                    'syntax-prism': {{ js_url_array('syntax-prism') }},
-                    'wiki-compat-tables': {{ js_url_array('wiki-compat-tables') }}
+                    'syntax-prism': [{% javascript 'syntax-prism' %}]
```
Comment (jwhitlock) [inline templates/includes/config.html]: Need a comma ``,`` at the end of the ``'syntax-prism'`` line

Comment (openjck) [inline templates/includes/config.html]: Yep, great catch

File: compressors/cleancss.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from pipeline.compressors import SubProcessCompressor
+
+class CleanCSSCompressor(SubProcessCompressor):
+    def compress_css(self, css):
+        return self.execute_command('cleancss', css)
```
Comment (openjck) [inline compressors/cleancss.py]: Not sure a new *compressors* directory is the best place for this. Open to suggestions!

Comment (stephaniehobson) [inline compressors/cleancss.py]: What about postprocessors?

Comment (stephaniehobson) [inline compressors/cleancss.py]: Or is this about having a top level directory at all and not the name?

Comment (openjck) [inline compressors/cleancss.py]: Mostly organization, yeah. I'm thinking back to that email thread from a while back about repository organization and I'm not sure where something like this fits in.

Comment (jwhitlock) [inline compressors/cleancss.py]: I vote ``/kuma/core/pipeline.py``.

Comment (openjck) [inline compressors/cleancss.py]: Updated!

**Comment by openjck** [comment]
Awesome!

**Comment by openjck** [comment]
I notice one issue with CKEditor. Looking into it...

**Comment by brhom300** [comment]
https://github.com/stephaniehobson/kuma/commit/524448e6fe6a451bd91147666475c2a01bce980c

**Comment by openjck** [comment]
Superseded by #3580 with squashed commits.

**Pull request closed without merging**